src/Sulu/Bundle/PageBundle/Admin/PageAdmin.php

```diff
@@ -116,9 +125,20 @@ public function configureRoutes(RouteCollection $routeCollection): void
             'sulu_admin.delete' => [
                 'display_condition' => '(!_permissions || _permissions.delete) && url != "/"',
             ],
-            'sulu_page.edit' => [
-                'copy_locale_display_condition' => '(!_permissions || _permissions.edit)',
-                'publish_display_condition' => '(!_permissions || _permissions.live)',
+            'sulu_admin.dropdown' => [
+                'label' => $this->translator->trans('sulu_admin.edit', [], 'admin'),
```

Comment:
Normally we give in this kind of configuration only translation keys and translate it in the frontend e.g.: https://github.com/sulu/sulu/blob/f1aabb71cb2590115b7ff4771b4a8f81574d9675/src/Sulu/Bundle/PageBundle/DependencyInjection/SuluPageExtension.php#L86

So would prefer todo this also here to set only the translation key and use the translator in the dropdown component.

Reply:
Seems like we already are inconsistent here... AFAIK we said in the beginning that we translate as much as possible on the server from the `/admin/config` request. This would have advantage that we have less work todo on the client, and this happens on the server only once per session. If you check e.g. the `teaser` stuff being returned from it or the occurence of the `sulu_admin.toggler` you'll see that this has already been translated on the server. I think the field type options are the exception here (or did you find any other examples?), because the translator is not available in the Extensions...

If we revoke that decision and say we always translate on the client, then we also have to fix that in other places, e.g. the teasers and other options on the route.

Reply:
@danrot Ok I would see it the other way around to translate it in JS so the PHP Developer has less todo 😄. On the RouteBuilder and Navigation they also need only to give translation keys so currently it seems in the Admin Class only the ToolbarActions need the translator.

/cc @sulu/core-team 

For `/admin/config` not sure but thats a request which a normal developer does not extend and when then add new configs where it is their choice to translate it in php or js.